### ISSUE TITLE
Shortcut key support for sending the tab into the latest session list

### ISSUE THREAD

**Issue by mustafa519** [one-tab-group/onetab.group#16]
Hello,

Thank you for the amazing extension. Just figured it out there is an extension which supports tab groups like I imagined.

Well, I was using those below shortcuts on OneTab and I feel lack of them on OneTab Gruop.

<img width="683" alt="image" src="https://user-images.githubusercontent.com/4794086/193718630-6e243580-b614-48d5-ab93-449629d90082.png">

And I still don't know how the sync works. I signed out with my Google Account, however, I am not sure how it's going to sync.

Thanks for the amazing work,
Mustafa.

**Comment by xiaoluoboding** [comment]
Hi, @mustafa519 ,

One Tab Group will support syncing your data in the future. And now only support syncing the settings with Google Account.

BTW, the shortcuts feature is on the way, Thanks for the suggestion.

**Comment by mustafa519** [comment]
Thanks for the explanation. I am still curious about where is the data. I mean I don't know how my Google account storage my data and where.

By the way, will we be able to change the session order? I wanted to move the sessions with custom order, I couldn't.

**Comment by xiaoluoboding** [comment]
Already supported in version 0.4.6, check it out.

**Comment by mustafa519** [comment]
Hey @xiaoluoboding,

Thanks for the efforts.

Well, when I tried the shortcut I lost all my saved sessions. They all disappeared suddenly, and I lost my session, so I had to login again. After losing the old session lists, it seems it's working normally. Is there a way for restoring them? I have lost approximately 400 tabs :/ Maybe, it happened because of the new update of the add-on, not sure, but it disappeared when I send the tab by the shortcut.

There are a few feedbacks that I need to share additionally:

1. I need the behavior like original onetab addon which lets me send the current tab into the latest session list rather than creating a new session.
2. "Open One Tab Group" shortcut opens up a new tab rather than focusing the opened tab already.
3. There is no an option to remove tab group from the list when I open a specific tab group in the session. I need to delete manually after open the group.
4. When I move a tab into a tabgroup in a session list, it seems working, however, when I open the group up the tabs are opening separately without respect to the pattern that I had set in the session list, like standalone tabs, without having a group.
